.changeset/nervous-pets-sleep.md

```diff
@@ -0,0 +1,140 @@
+---
+'@primer/react': major
+---
+
+The `CheckboxGroup` and `RadioGroup` components will be used to deprecate the `ChoiceFieldset` component. The deprecation of `ChoiceFieldset` also allows us to deprecate `ChoiceInputField`.
```

Comment:
If it's a quick one, can we add the heading in? Happy to do it in the final changelog too, so no biggie

```
### ChoiceFieldset and ChoiceFieldset
```